src/amo/containers/Home.js

```diff
@@ -34,6 +34,21 @@ export class HomePageBase extends React.Component {
     const { i18n } = this.props;
     return (
       <div className="HomePage">
+        <div className="HomePage-welcome">
+          <p className="HomePage-welcome-text">
+            {i18n.gettext(dedent`Extensions are special features you can add to Firefox.
+            Themes let you change your browser's appearance.`)}
+          </p>
+          <div className="HomePage-welcome-links">
+            <Link className="HomePage-extensions-link" to="/extensions/">
```

Comment:
Yeah, agreed. I think react-router used to have a feature like `routeFor` or something way back but they ditched it. I'm not too bothered by it as URLs shouldn't change, just a thought 😄